package.json

```diff
@@ -4,28 +4,46 @@
   "description": "NZVirtual Flight Tracker API service",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "nodemon src/index.js"
+    "test": "yarn format && yarn lint",
+    "format": "prettier-eslint —eslint-config-path $PWD/.eslintrc --write '$PWD/**/*.js' && eslint --fix .",
+    "babel": "babel-node --presets @babel/preset-env",
+    "lint": "eslint . -ext src/**.js",
+    "dev": "nodemon --exec yarn babel -- src/index.js",
+    "build": "babel src -d dist",
+    "prod": "node dist/index.js"
+  },
+  "husky": {
+    "pre-commit": "yarn format && yarn lint"
   },
   "keywords": [],
-  "author": "Daniel A. Hawton <daniel@nzvirtual.org>",
+  "author": "Daniel A. Hawton <daniel@nzvirtual.org> and Rahul A. Parkar <rahul@nzvirtual.org>",
   "license": "MIT",
   "dependencies": {
+    "basic-auth": "^2.0.1",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "express-basic-auth": "^1.2.0",
     "jsonwebtoken": "^8.5.1",
     "mysql2": "^2.1.0",
     "sequelize": "^5.21.11"
   },
   "devDependencies": {
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.2",
+    "@babel/node": "^7.10.1",
+    "@babel/preset-env": "^7.10.2",
+    "babel-preset-minify": "^0.5.1",
     "eslint": "6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
     "nodemon": "^2.0.4",
-    "prettier": "^2.0.5"
+    "prettier": "^2.0.5",
+    "prettier-eslint": "^10.1.1",
```

Comment:
What this?

Reply:
Can you not just have it running from within ESLint, as you seem to be attempting to do using https://prettier.io/docs/en/integrating-with-linters.html#disable-formatting-rules

Reply:
It has both, built-in formatting with husky, as well as with linting and it provides tools to VSCode